src/app/modules/item/pages/item-details/item-details.component.html

```diff
@@ -22,18 +22,22 @@
         Content
       </a>
       <a
+        *ngIf="!(watchedGroup$ | async) || !!(watchedGroup$ | async) && itemData.item.permissions.canWatch !== 'none' || !!(watchedGroup$ | async) && itemData.item.permissions.canWatch === 'none' && !contentTab.isActive"
```

Comment:
The condition looks good but can be simplified to 
```
!(watchedGroup$ | async) || itemData.item.permissions.canWatch !== 'none' || !contentTab.isActive"
```
easier 🙂

Reply:
Thank you, fixed